dovpanda/core.py

```diff
@@ -206,3 +206,20 @@ def dont_append_with_loop(arguments):
         ledger.tell('dont append or concat dfs iteratively. '
                     'it is a better practice to first create a list of dfs. '
                     'and then <code>pd.concat(list_of_dfs)</code> in one go')
+
+
+@ledger.add_hint('Series.str.split', 'post')
+def suggest_expand(res, arguments):
+    expand = arguments.get('expand')
+    pat = arguments.get('pat')
+    if expand:
+        return
+    if hasattr(res, 'name'):
+        col = res.name
+    else:
+        col = 'series'
+    if not expand:
```

Comment:
This condition is redundant, you verified expand is false before

Reply:
+1